src/v/cluster/members_backend.cc

```diff
@@ -43,6 +45,180 @@
 #include <vector>
 
 namespace cluster {
+namespace {
+struct node_replicas {
+    size_t allocated_replicas;
+    size_t max_capacity;
+};
+
+using node_replicas_map_t = absl::node_hash_map<model::node_id, node_replicas>;
+static absl::node_hash_map<model::node_id, node_replicas>
```

Comment:
since you are inside an anonymous namespace you don't need to mark the function as file scope to prevent it from becoming an exported symbol